workshops/geometric_pattern/README.md

```diff
@@ -561,17 +596,15 @@ This code says that if the user presses a key with the `keyCode` 115 or 83 (whic
 
 The `return false;` at the end of the function tells the browser to ignore any default behavior that might be associated with the pressed key (ex. if you press enter, the browser thinks that you're trying to submit a form by default -- this prevents that).
 
-### Making It Live
-
-Make sure all of your files are saved. Then, head on over to the terminal in Cloud9 by pressing `alt+t` and type the following commands (pressing Enter after each one):
+Now let's commit these changes:
 
 - `git add --all`
-- `git commit -m "Create a geometric pattern using p5.js"`
+- `git commit -m "Add save keybinding to geometric_pattern"`
 - `git push`
 
-Then, enter your GitHub username and password (careful here, as password will not be displayed).
+### Making It Live
 
-Congratulations! Your pattern is now live on `USERNAME.github.io/geometric_pattern/` (replace `USERNAME` with your own GitHub username!)
+Congratulations! Ever since your first commit, your pattern has been online at `USERNAME.github.io/geometric_pattern/` (replace `USERNAME` with your own GitHub username!).
```

Comment:
Yeah, me too. Maybe if it was reworded it would be ok...

On Tue, 16 Aug 2016 11:44 pm Zach Latta <notifications@github.com> wrote:

> In workshops/geometric_pattern/README.md
> <https://github.com/hackclub/hackclub/pull/891#discussion_r74936468>:
>
> >
> > -Congratulations! Your pattern is now live on `USERNAME.github.io/geometric_pattern/` <http://USERNAME.github.io/geometric_pattern/> (replace `USERNAME` with your own GitHub username!)
> > +Congratulations! Ever since your first commit, your pattern has been online at `USERNAME.github.io/geometric_pattern/` <http://USERNAME.github.io/geometric_pattern/> (replace `USERNAME` with your own GitHub username!).
>
> I'm cool with anything, just worried that "you could share with your
> friend or family!" comes of as cheesy and something an out-of-touch teacher
> would say.
> … <#m_-1878161988695864675_>
> On Tue, Aug 16, 2016 at 6:38 AM, Harrison Shoebridge < ***@***.***> wrote:
> In workshops/geometric_pattern/README.md <#891 (comment)
> <https://github.com/hackclub/hackclub/pull/891#discussion_r74935464>>: >
> > -Congratulations! Your pattern is now live on `
> USERNAME.github.io/geometric_pattern/`
> <http://USERNAME.github.io/geometric_pattern/> <
> http://USERNAME.github.io/geometric_pattern/> (replace `USERNAME` with
> your own GitHub username!) > +Congratulations! Ever since your first
> commit, your pattern has been online at `
> USERNAME.github.io/geometric_pattern/`
> <http://USERNAME.github.io/geometric_pattern/> <
> http://USERNAME.github.io/geometric_pattern/> (replace `USERNAME` with
> your own GitHub username!). I was thinking more of a textual thing like: If
> you wanted you could share this with your friends or family! But I like the
> idea of a visual phone too, potentially more really. — You are receiving
> this because you were assigned. Reply to this email directly, view it on
> GitHub <
> https://github.com/hackclub/hackclub/pull/891/files/2ee60249ee4804ba05b9219acfb0869631954216#r74935464>,
> or mute the thread <
> https://github.com/notifications/unsubscribe-auth/AA8j-Flhqf8phaCjkAnhyN6AKIALO8CEks5qgb1tgaJpZM4Jgmn->
> .
> -- Zach Latta https://zachlatta.com <http://zachlatta.com>
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/hackclub/hackclub/pull/891/files/2ee60249ee4804ba05b9219acfb0869631954216#r74936468>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AIBJSuhn6TixSZyyhZREncQZhWmZXjm9ks5qgb6igaJpZM4Jgmn->
> .
>